client/admin/notificationUtils/notificationUtils.jsx

```diff
@@ -0,0 +1,19 @@
+const React = require('react');
+const createClass = require('create-react-class');
+
+const NotificationLookup = require('./notificationLookup/notificationLookup.jsx');
+const NotificationAdd = require('./notificationAdd/notificationAdd.jsx');
```

Comment:
This file doesn't exist. Does it need to be committed?

Reply:
Yes - that's in the next commit.